src/poetry/console/commands/version.py

```diff
@@ -29,7 +29,14 @@ class VersionCommand(Command):
             optional=True,
         )
     ]
-    options = [option("short", "s", "Output the version number only")]
+    options = [
+        option("short", "s", "Output the version number only"),
+        option(
+            "dry-run",
+            None,
+            "Outputs the version without updating the pyproject.toml file",
```

Comment:
```suggestion
            "Do not update pyproject.toml file",
```

This might suffice. Because technically we do not output the version but the version bump log.